x/nft/keeper/msg_server.go

```diff
@@ -51,6 +51,7 @@ func (m msgServer) IssueDenom(goCtx context.Context, msg *types.MsgIssueDenom) (
 	return &types.MsgIssueDenomResponse{}, nil
 }
 
+// nolint: dupl
```

Comment:
perhaps a TODO to refactor this if it'll make it better -- I guess the linter error is about the nearly duplicate "EmitEvents" bit which could perhaps be abstracted out?

Reply:
Yeah. Because the `msg` is of different type with almost similar fields (but they do not implement same interface). I'm not sure how we can abstract this out. Is there something similar to Rust macros in Go?

I'll add a `TODO` for now.

Reply:
Done